src/sentry/mail/adapter.py

```diff
@@ -112,30 +110,18 @@ def get_sendable_user_objects(project):
         user_ids = project.member_set.values_list("user_id", flat=True)
         users = user_service.get_many(filter=dict(user_ids=list(user_ids)))
 
-        if should_use_notifications_v2(project.organization):
-            controller = NotificationController(
-                recipients=users,
-                project_ids=[project.id],
-                organization_id=project.organization_id,
-                provider=ExternalProviderEnum.EMAIL,
-                type=NotificationSettingEnum.ISSUE_ALERTS,
-            )
-            return controller.get_notification_recipients(
-                type=NotificationSettingEnum.ISSUE_ALERTS,
-                actor_type=ActorType.USER,
-            )[ExternalProviders.EMAIL]
-
-        accepting_recipients = NotificationSetting.objects.filter_to_accepting_recipients(
-            project, users
+        # Do we need to use a notification service here?
```

Comment:
should this be marked as a TODO?

Reply:
@jangjodi Not quite a TODO because I'm not sure we need it. I can change to a TODO if you think that makes sense.

Reply:
Maybe you could mark the question as TODO so it's easier to find and be re-visited? I'm just not sure if anyone's going to feel inclined to try to answer the question and it might get lost